### PR TITLE
Added script for converting energy profiles with arbitrary resolution…

### PR DESCRIPTION
**Pull request by spyrostz**
… to power profiles with market slot length resolution.

### CODE REVIEW

File: src/d3a/resources/convert_energy_to_power_profile.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        if time in input_profile.keys():
+            current_val = input_profile[time]
+        out_profile[time] = current_val
+
+    return out_profile
+
+
+def convert_energy_profile_to_power(input_profile, output_file):
+    profile = _readCSV(input_profile)
+    _eval_time_period_consensus(profile)
```
Comment (hannesdiedrich) [inline src/d3a/resources/convert_energy_to_power_profile.py]: Just minor. But as this is a preprocessing step, the user might want to not care about the time period consensus.
But i dont have a strong opinion.

File: src/d3a/resources/convert_energy_to_power_profile.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        # Average market slot values
+        iter_duration = duration(minutes=0)
+        averaged_value = 0
+        while iter_duration < GlobalConfig.slot_length:
+            averaged_value += profile[k + iter_duration]
+            iter_duration += duration(minutes=1)
+        averaged_value /= GlobalConfig.slot_length.minutes
+        output_dict[k] = averaged_value
+
+    def convert_energy_to_power(e):
```
Comment (hannesdiedrich) [inline src/d3a/resources/convert_energy_to_power_profile.py]: Minor: Could be moved outside of the parent function.

Comment (spyrostz) [reply src/d3a/resources/convert_energy_to_power_profile.py]: Will do, thanks!

File: src/d3a/resources/convert_energy_to_power_profile.py (hunk 1 above)
Comment (spyrostz) [reply src/d3a/resources/convert_energy_to_power_profile.py]: The thing is that Fatma does, she wanted to reinterpolate the data in 15 minutes intervals (for the market slot length). If you agree I can refactor this function to accept the time period as an explicit input argument, and not to rely on the const settings for that.

**Pull request merged**